e2e-tests/package.json

```diff
@@ -35,12 +35,13 @@
     "eslint": "8.57.1",
     "eslint-plugin-filenames": "1.3.2",
     "ioredis": "5.6.1",
-    "otplib": "12.0.1",
+    "otplib": "^12.0.1",
     "prettier": "3.5.3",
     "typescript": "5.8.3"
   },
   "dependencies": {
     "@azure/identity": "4.9.1",
+    "@backstage/catalog-model": "^1.7.4",
```

Comment:
I think we don't need to import this library. You are importing this lib only to type a field as far as I could see